lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py

```diff
@@ -61,10 +61,8 @@ def some_event_function():
 RELATION_KEYS = [
     "consent_url",
     "error_url",
-    "index_url",
     "login_url",
     "oidc_error_url",
-    "registration_url",
     "default_url",
```

Comment:
If you remove the index page, then you should update the `default_url` as well (it used to be the `index` page)